static/src/js/components/FormNavigation/FormNavigation.jsx

```diff
@@ -142,6 +142,7 @@ const FormNavigation = ({
             {
               !isTemplate && (
                 <Dropdown.Item
+                  disabled={errors.length > 0}
```

Comment:
Here we should also add a tooltip describing why the button is disabled

Reply:
Agreed. Because I'm encountering this issue and I'm not sure why: 
<img width="1357" alt="Screenshot 2024-07-17 at 3 13 34 PM" src="https://github.com/user-attachments/assets/ea8fef70-1d13-4049-8e42-fe501209f531">

Reply:
^^This holds true both before and after I click 'save'

Reply:
Yeah there is a reason for that, it is due to the schemas not being in sync.   That will be fixed with deep's ticket.  If you try creating a umm-var instead of a collection where the schemas are the same the button enables properly.